diff -u -r /usr/share/lorax/templates.d/80-rhel/live/config_files/x86/grub.conf live-tmpl/live/config_files/x86/grub.conf
--- /usr/share/lorax/templates.d/80-rhel/live/config_files/x86/grub.conf	2019-03-02 04:33:14.000000000 +0900
+++ live-tmpl/live/config_files/x86/grub.conf	2019-12-09 13:34:17.760273821 +0900
@@ -5,9 +5,9 @@
 hiddenmenu
 title Start @PRODUCT@ @VERSION@
 	findiso
-	kernel @KERNELPATH@ @ROOT@ rd.live.image quiet
+	kernel @KERNELPATH@ @ROOT@ rd.live.image quiet @EXTRA@
 	initrd @INITRDPATH@
 title Test this media & start @PRODUCT@ @VERSION@
 	findiso
-	kernel @KERNELPATH@ @ROOT@ rd.live.image rd.live.check quiet
+	kernel @KERNELPATH@ @ROOT@ rd.live.image rd.live.check quiet @EXTRA@
 	initrd @INITRDPATH@
diff -u -r /usr/share/lorax/templates.d/80-rhel/live/config_files/x86/grub2-efi.cfg live-tmpl/live/config_files/x86/grub2-efi.cfg
--- /usr/share/lorax/templates.d/80-rhel/live/config_files/x86/grub2-efi.cfg	2019-03-02 04:33:14.000000000 +0900
+++ live-tmpl/live/config_files/x86/grub2-efi.cfg	2019-12-09 13:32:54.545668362 +0900
@@ -21,16 +21,16 @@
 
 ### BEGIN /etc/grub.d/10_linux ###
 menuentry 'Start @PRODUCT@ @VERSION@' --class fedora --class gnu-linux --class gnu --class os {
-	linuxefi @KERNELPATH@ @ROOT@ rd.live.image quiet
+	linuxefi @KERNELPATH@ @ROOT@ rd.live.image quiet @EXTRA@
 	initrdefi @INITRDPATH@
 }
 menuentry 'Test this media & start @PRODUCT@ @VERSION@' --class fedora --class gnu-linux --class gnu --class os {
-	linuxefi @KERNELPATH@ @ROOT@ rd.live.image rd.live.check quiet
+	linuxefi @KERNELPATH@ @ROOT@ rd.live.image rd.live.check quiet @EXTRA@
 	initrdefi @INITRDPATH@
 }
 submenu 'Troubleshooting -->' {
 	menuentry 'Start @PRODUCT@ @VERSION@ in basic graphics mode' --class fedora --class gnu-linux --class gnu --class os {
-		linuxefi @KERNELPATH@ @ROOT@ rd.live.image nomodeset quiet
+		linuxefi @KERNELPATH@ @ROOT@ rd.live.image nomodeset quiet @EXTRA@
 		initrdefi @INITRDPATH@
 	}
 }
diff -u -r /usr/share/lorax/templates.d/80-rhel/live/config_files/x86/isolinux.cfg live-tmpl/live/config_files/x86/isolinux.cfg
--- /usr/share/lorax/templates.d/80-rhel/live/config_files/x86/isolinux.cfg	2019-03-02 04:33:14.000000000 +0900
+++ live-tmpl/live/config_files/x86/isolinux.cfg	2019-12-09 13:33:58.858211873 +0900
@@ -61,13 +61,13 @@
 label linux
   menu label ^Start @PRODUCT@ @VERSION@
   kernel vmlinuz
-  append initrd=initrd.img @ROOT@ rd.live.image quiet
+  append initrd=initrd.img @ROOT@ rd.live.image quiet @EXTRA@
 
 label check
   menu label Test this ^media & start @PRODUCT@ @VERSION@
   menu default
   kernel vmlinuz
-  append initrd=initrd.img @ROOT@ rd.live.image rd.live.check quiet
+  append initrd=initrd.img @ROOT@ rd.live.image rd.live.check quiet @EXTRA@
 
 menu separator # insert an empty line
 
@@ -83,7 +83,7 @@
 	@PRODUCT@ @VERSION@.
   endtext
   kernel vmlinuz
-  append initrd=initrd.img @ROOT@ rd.live.image nomodeset quiet
+  append initrd=initrd.img @ROOT@ rd.live.image nomodeset quiet @EXTRA@
 
 label memtest
   menu label Run a ^memory test
diff -u -r /usr/share/lorax/templates.d/80-rhel/live/efi.tmpl live-tmpl/live/efi.tmpl
--- /usr/share/lorax/templates.d/80-rhel/live/efi.tmpl	2019-03-02 04:33:14.000000000 +0900
+++ live-tmpl/live/efi.tmpl	2019-12-09 13:58:40.249773926 +0900
@@ -45,6 +45,7 @@
     replace @KERNELPATH@ /${kdir}/vmlinuz ${eficonf}
     replace @INITRDPATH@ /${kdir}/initrd.img ${eficonf}
     replace @ISOLABEL@ '${isolabel}' ${eficonf}
+    replace @EXTRA@ '${extra_boot_args}' ${eficonf}
     %if disk:
         replace @ROOT@ root=live:LABEL=ANACONDA ${eficonf}
     %else:
diff -u -r /usr/share/lorax/templates.d/80-rhel/live/x86.tmpl live-tmpl/live/x86.tmpl
--- /usr/share/lorax/templates.d/80-rhel/live/x86.tmpl	2019-03-02 04:33:14.000000000 +0900
+++ live-tmpl/live/x86.tmpl	2019-12-09 13:57:41.636447733 +0900
@@ -48,6 +48,7 @@
 replace @VERSION@ ${product.version} ${BOOTDIR}/grub.conf ${BOOTDIR}/isolinux.cfg ${BOOTDIR}/*.msg
 replace @PRODUCT@ '${product.name}'  ${BOOTDIR}/grub.conf ${BOOTDIR}/isolinux.cfg ${BOOTDIR}/*.msg
 replace @ROOT@ 'root=live:CDLABEL=${isolabel|udev}' ${BOOTDIR}/isolinux.cfg
+replace @EXTRA@ '${extra_boot_args}' ${BOOTDIR}/isolinux.cfg
 
 ## install kernels
 mkdir ${KERNELDIR}
